email/1.3/modules/ROOT/edits/email-send.adoc

```diff
@@ -0,0 +1,363 @@
+= Send Emails with Email Connector Examples - Mule 4
+:keywords: email, connector, configuration, smtp, send, smtps
+:page-aliases: connectors::email/email-send.adoc
+
+Anypoint Connector for Email (Email Connector) can send messages over SMTP and SMTPS servers by using the *Send* operation. The following examples show how to configure Email Connector to send emails and attachments, in both Anypoint Studio and the XML editor:
```

Comment:
Minor point, but I wonder if we should reword this to put the audience into the action:
```suggestion
Use Anypoint Connector for Email (Email Connector) to send messages over SMTP and SMTPS servers by using the *Send* operation. The following examples show how to configure Email Connector to send emails and attachments, in both Anypoint Studio and the XML editor:
```